QUARKUS-728.md

```diff
@@ -7,8 +7,11 @@ The goal is to verify OpenJDK 17 integration
 ## Scope of the testing
 Test development will focus on  
  - New specifict scenarios will be created to cover the integration of new objects as “sealed” or “records” classes with several extensions as Hibernate or RESTEasy.
- - Current scenarios must work with Java 17
+ - Extended Quarkus test suite scenarios must work with Java 17
```

Comment:
What do you mean by this?

Reply:
Basically, that native build is in the preview stage, so its going to be covered by extended Quarkus pipelines.